### PR TITLE
fix mangled url

### PR DESCRIPTION
**Pull request by shubhgng**
Crash help text has mangled URL #61

### CODE REVIEW

File: pkl-core/src/main/java/org/pkl/core/runtime/VmExceptionRenderer.java [hunk 1]
```diff
@@ -59,7 +59,8 @@ private void renderBugException(VmBugException exception, StringBuilder builder)
         .append("Cmd+Double-click the link below to open an issue.\n")
         .append(
             "Please copy and paste the entire error output into the issue's description, provided you can share it.\n\n")
-        .append("https://github.com/apple/pkl/issues/new");
+        .append("https://github.com/apple/pkl/issues/new")
+        .append("\n\n");
```
Comment (holzensp) [inline pkl-core/src/main/java/org/pkl/core/runtime/VmExceptionRenderer.java:63]: Yeah, except maybe keep it brief
```suggestion
        .append("https://github.com/apple/pkl/issues/new\n\n");
```

**Review by holzensp — APPROVED** [review]
Awaiting a CI config snag, but LGTM (see nit)

File: pkl-core/src/main/java/org/pkl/core/runtime/VmExceptionRenderer.java (hunk 1 above)
Comment (shubhgng) [reply pkl-core/src/main/java/org/pkl/core/runtime/VmExceptionRenderer.java]: @holzensp sure I have made the changes.

**Comment by stackoverflow** [comment]
Some tests are broken and should be fixed before we can merge this.

**Comment by holzensp** [comment]
Shooting from the hip, but I'd imagine _all_ of these tests have become flakey since the Gradle bump. We should kick this off again after that's reverted.

**Comment by shubhgng** [comment]
Much appreciated, thanks

**Pull request merged**